src/sif/autocost.cc

```diff
@@ -232,7 +232,8 @@ class AutoCost : public DynamicCost {
   virtual Cost EdgeCost(const baldr::DirectedEdge* edge,
                         const graph_tile_ptr& tile,
                         const uint32_t seconds,
-                        uint8_t& flow_sources) const override;
+                        uint8_t& flow_sources,
+                        const uint32_t travel_time_seconds) const override;
```

Comment:
Usually immutable parameters go first, and mutable ones go last. Could you change the order please? and below